src/Controller/MailPreviewController.php

```diff
@@ -128,7 +128,7 @@ public function email($name, $method)
         $this->set('title', $humanName);
         $this->set('email', $email);
         $this->set('plugin', $plugin);
-        $this->set('part', $this->findPreferredPart($email, $this->request->getQuery('part')));
+        $this->set('part', $this->findPreferredPart($email, $this->getRequest()->getQuery('part')));
```

Comment:
Controller's don't need to use `getRequest()` as the request is a protected property.